Apigee/Mint/DeveloperRatePlan.php

```diff
@@ -359,14 +366,30 @@ public function isCancelable()
 
     /* Setters */
 
-    public function setStartDate($start_date)
+    /**
+     * @param string $start_date Date string.
+     * @param string $format Format of the date string. Any acceptable format by date(), default is the Edge API
+     * format, 'Y-m-d H:i:s'.
+     * @param string|null $timezone Timezone of the date string. One of the supported timezone names or
+     * an offset value (+0200). Default is the timezone of the organisation.
+     */
+    public function setStartDate($start_date, $format = self::DATE_FORMAT, $timezone = null)
```

Comment:
Passing DateTime objects would be better because we can get rid of these "dummy" parameters, but we must keep backward compatibility. That is why I also added $timezone as a string rather than a DateTimezone object. Maybe in the next major release, these will be fixed.